modules/update-preparing-forced-update-risk.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing_for_updates/updating-cluster-prepare.adoc
+
+:_content-type: CONCEPT
+[id="update-preparing-forced-update-risk"]
+= Assessing the risk of forced updates
+
+If there is no other way to update a cluster, you can "force" an update by using the `--force` option in the CLI, which allows you to forcefully upgrade the cluster even when upgrade release image validation fails and the cluster is reporting errors.
```

Comment:
If we're talking about updates when _not_ referring to `oc adm upgrade`, `Upgradeable=false`, or other contexts where the word "upgrade" is hard coded into something, we should use "update" instead of "upgrade":
```suggestion
If there is no other way to update a cluster, you can "force" an update by using the `--force` option in the CLI, which allows you to forcefully update the cluster even when update release image validation fails and the cluster is reporting errors.
```